tools/generate-mocks/generate-mocks.ts

```diff
@@ -32,15 +32,16 @@ export class GenerateMocks {
     const outputPathNormalized = path.normalize(outputPaths.base);
     const classMap = new Map<string, string[]>();
     const exportedProviders: ComponentMetaData[] = [];
-    const assertedTypesMap = new Map<string, string>();
-    const exportedTypes = await this.getExportedTypes(inputPath, assertedTypesMap);
+    const aliasesMap = new Map<string, string>();
+    const exportedTypesAndAliases = await this.getExportedTypesAndAliases(inputPath);
```

Comment:
Consider>

```suggestion
    const exportedTypesWithAliases = await this.getExportedTypesWithAliases(inputPath);
```

The way around sounds like it is two separate things 😄